extensions/amp-story/1.0/media-pool.js

```diff
@@ -180,27 +167,6 @@ export class MediaPool {
     /** @private {?Array<!AmpElement>} */
     this.ampElementsToBless_ = null;
 
-    /** @const {!Object<string, (function(): !PoolBoundElementDef)>} */
-    this.mediaFactory_ = {
```

Comment:
These factories don't vary much, and it's near impossible that we'll have to support anything other than `audio` and `video` in the future. Removed the factories and folded in below.